src/github/client/github-client.ts

```diff
@@ -65,4 +95,26 @@ export class GitHubClient {
 			proxy: false
 		};
 	};
+
+	protected async graphql<T>(query: string, config: AxiosRequestConfig, variables?: Record<string, string | number | undefined>): Promise<AxiosResponse<GraphQlQueryResponse<T>>> {
```

Comment:
love that you moved this to the superclass :+1: 

We need to do the same with all other base function like get, post, put, delete, etc.  For a later PR :)